src/middleware.ts

```diff
@@ -1129,12 +1129,16 @@ const REDIRECTS: Redirect[] = [
     from: '/product/performance/event-detail/',
     to: '/product/sentry-basics/concepts/key-terms/tracing/trace-view/',
   },
+  {
+    from: '/product/performance/event-detail/',
+    to: '/product/sentry-basics/concepts/key-terms/tracing/trace-view/',
```

Comment:
```suggestion
    to: '/concepts/key-terms/tracing/trace-view/',
```

correct path doesn't have /product/sentry-basics

page should go to this: https://docs.sentry.io/concepts/key-terms/tracing/trace-view/